app/views/home/index.html.erb

```diff
@@ -15,13 +15,13 @@
       </ul>
     </div>
   <% else %>
-    <div class="browse module">
-      <h3 class="heading"><%= ts('Find your favorites') %></h3>
+    <nav aria-label="<%= t(".media_navigation_label") %>" class="browse module">
+      <h3 class="heading"><%= t(".find_your_favorites") %></h3>
       <% if logged_in? %>
-        <p class="note"><%= ts('Browse fandoms by media or favorite up to %{maximum} tags to have them listed here!', maximum: ArchiveConfig.MAX_FAVORITE_TAGS) %></p>
+        <p class="note"><%= t(".browse_or_favorite", maximum: ArchiveConfig.MAX_FAVORITE_TAGS) %></p>
```

Comment:
I think we need to pass `ArchiveConfig.MAX_FAVORITE_TAGS` as `count` so Translation can have multiple versions of this sentence ready in case we ever change `MAX_FAVORITE_TAGS`.

Reply:
I think sarken means this: https://guides.rubyonrails.org/i18n.html#pluralization

Reply:
Thank you! Hashtag today-I-learned :D

Reply:
Yup, that's what I meant! 

When we use `count`, we generally define `one:` and `other:` English translations to go with it. It's _pretty_ unlikely we'll ever drop `MAX_FAVORITE_TAGS` down to one, but if you could update `en.yml` to have both just to be safe, this will be ready to merge:

```
    index:
      browse_or_favorite:
        one: Browse fandoms by media or favorite up to %{count} tag to have them listed here!
        other: Browse fandoms by media or favorite up to %{count} tags to have them listed here!
```

Reply:
Ahaha, I was coming here to nitpick that very thing :D 

(You have to admit that "favourite up to 1 tag" sounds pretty sad though, and the "have it listed here" only adds to the desolation...)

Reply:
It really does. Something like "or favorite a tag/choose a favorite tag" sounds a bit less sad, or at least less awkward, but it's probably not worth getting bogged down in wording that shouldn't see the light of day unless Catastrophe™  strikes. (Unless you want to! I don't mind workshopping wording; I just don't want you to feel obligated.)